stream-chat-android-docs/build.gradle

```diff
@@ -1,23 +1,17 @@
 import com.getstream.sdk.chat.Dependencies
-import com.getstream.sdk.chat.Configuration
 
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'org.jlleitschuh.gradle.ktlint'
 
-android {
-    compileSdkVersion Configuration.compileSdkVersion
+apply from: "${rootDir}/scripts/android.gradle"
 
+android {
     defaultConfig {
         applicationId "io.getstream.chat.docs"
```

Comment:
Do we need `gitVersioner` here? Seems that we don't publish this module, so I removed it.